src/deployment/azuredeploy.bicep

```diff
@@ -80,29 +75,16 @@ module operationalInsights 'bicep-templates/operational-insights.bicep' = {
   }
 }
 
-module linuxServerFarm 'bicep-templates/server-farms.bicep' = {
-  name: 'linux-server-farm'
+module serverFarm 'bicep-templates/server-farms.bicep' = {
+  name: 'server-farm'
   params: {
     server_farm_name: name
     owner: owner
     location: location
-    use_windows: false
-    create: true
```

Comment:
I know you've tested, but we don't need the `create : true` property?

Reply:
Nope!